### ISSUE TITLE
2.02 RPT error WL2_zoneRestrictionHandleServer.sqf, line 52

### ISSUE THREAD

**Issue by Dwarden** [Gamer-Dad/warlordsredux.altis#11]
server's RPT error in 2.02
```
2022/02/19, 22:30:37 getVariable called on null object
2022/02/19, 22:30:37  ➥ Context: 	[] L52 (mpmissions\__cur_mp.Altis\Functions\server\WL2_zoneRestrictionHandleServer.sqf)
	[] L52 (mpmissions\__cur_mp.Altis\Functions\server\WL2_zoneRestrictionHandleServer.sqf)

2022/02/19, 22:30:37 Error in expression <all BIS_fnc_WL2_syncedTime) >= _timeout || (_player getVariable "BIS_WL_zoneRest>
   Error position: <|| (_player getVariable "BIS_WL_zoneRest>
   Error Undefined behavior: waitUntil returned nil. True or false expected.
2022/02/19, 22:30:37 File mpmissions\__cur_mp.Altis\Functions\server\WL2_zoneRestrictionHandleServer.sqf..., line 52
```

**Comment by korbelz** [comment]
lowered zone timesouts back to default and tested this locally no errors. 

was unable to test on dedi server due to verison mismatch

**Comment by korbelz** [comment]
on Dedi server: I walked into, drove into and flew into killzones around towns was unable to repo this rpt file :(

using .57 killzone timers

**Comment by korbelz** [comment]
More testing on this on dedi server: Tested boats, UAVs, AI walking into zones, AI driving into zones, AI in my heli flying into zones, and flying off the map. Nothing I can think of will repo this bug. 

I have no clue what is causing this bug, the code looks fine. I slowed down the check and made the kill timers 1 second to make it more simple and less rpt spam. 

https://github.com/korbelz/WarlordsReduxMe.altis/commit/e7b3ce0f89717a3e1382f24b21fc63dcc7186258
